src/studylist/StudyListWithData.js

```diff
@@ -30,10 +34,28 @@ class StudyListWithData extends Component {
   static defaultStudyDateTo = new Date();
 
   componentDidMount() {
-    // TODO: Avoid using timepoints here
-    //const params = { studyInstanceUids, seriesInstanceUids, timepointId, timepointsFilter={} };
+    if (!this.props.server) {
```

Comment:
Can we put this behind `enableGoogleCloudAdapter` as well?

Reply:
done